pkg/artifactory/resource_artifactory_user.go

```diff
@@ -166,7 +178,7 @@ func resourceUserCreate(d *schema.ResourceData, m interface{}) error {
 
 	d.SetId(user.Name)
 	return resource.Retry(d.Timeout(schema.TimeoutCreate), func() *resource.RetryError {
-		result := &services.User{}
+		result := &User{}
```

Comment:
Nice catch. I would love to see how much more exposure we have to the jfrog client